src/helix-ui/elements/HXFileIconElement.less

```diff
@@ -28,3 +28,20 @@
     text-transform: uppercase;
   }
 }
+
+:host(:not([extension])) {
```

Comment:
We should probably invert the styling logic for this (centered by default, update only when `[extension]` is present)

```less
:host {
  // styles for centered icon
}
:host([extension]) {
  // reposition/restyle to account for extension
}
```